src/caselawclient/models/judgments.py

```diff
@@ -23,66 +23,101 @@
 JUDGMENT_STATUS_IN_PROGRESS = "In progress"
 
 
+class MarkLogicAttributeDict(TypedDict):
+    getter: Callable[[str], Union[str, bool]]
+    setter: None
+
+
 class CannotPublishUnpublishableJudgment(Exception):
     pass
 
 
 class Judgment:
+    MARKLOGIC_ATTRIBUTE_MAP: dict[str, MarkLogicAttributeDict]
+
+    values_store: dict[str, Union[str, bool]] = {}
+
+    # These type definitions make sure that downstream users of this class can rely on the types
+    uri: str
+    api_client: MarklogicApiClient
+    name: str
+    neutral_citation: str
+    court: str
+    judgment_date_as_string: str
+    is_published: bool
+    is_sensitive: bool
+    is_anonymised: bool
```

Comment:
Isn't it only `MARKLOGIC_ATTRIBUTE_MAP`, `values_store`, `uri` and `api_client` that are properties on the `Judgment` class now, and that therest are just keys in the `MARKLOGIC_ATTRIBUTE_MAP`?